docs/assets/css/anchorjs.css

```diff
@@ -6,21 +6,24 @@
     color: inherit !important;
     text-decoration: none !important; /* do not underline */
     }
-    
+
+#search-results .anchorjs-link {
+    right: 0;
```

Comment:
we should find a way out , this isn't desired as it is getting unnoticed mostly in this case.
The anchor should be placed just right to the heading. 🤔

Reply:
I agree. The other way I can think of, that would get us the desired result will be to change the position of the DOM elements in html source code. I tried to look for the html code but couldn't find it. What would you suggest ?